test-integration/task-plan/publish.coffee

```diff
@@ -6,29 +6,38 @@ TEACHER_USERNAME = 'teacher01'
 
 describe 'Assignment Publishing Tests', ->
 
-  @it 'Publishes a Reading with opensAt to tomorrow and deletes (idempotent)', ->
-    title = @freshId()
-
-    @login(TEACHER_USERNAME)
+  beforeEach ->
+    @title = freshId()
+    new User(@, TEACHER_USERNAME).login()
```

Comment:
Why would `displayProfile` need to already have the username? It seems like `user.displayProfile()` would just need to click on the `<div class="profile">Profile</div>` element, no?